server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -275,6 +275,10 @@ timestamp_precision=ms
 # Datatype: int
 # concurrent_query_thread=16
 
+# How many threads can concurrently run migration tasks. When <= 0, use CPU core number.
+# Datatype: int
+# migration_thread_num=16
```

Comment:
The migration tasks are not performance-critical, using 16 threads to execute this task is unsuitable.

Reply:
Changed to 2.